modules/nested-tabs/assets/scss/frontend.scss

```diff
@@ -243,7 +236,9 @@
 		}
 
 		> .e-con:not(.e-active) {
-			display: none;
```

Comment:
Without the display none, see what I'm getting
![image](https://user-images.githubusercontent.com/82999080/223703566-5cde53dc-92d8-4a55-8d27-7330ef26dcbf.png)

- Set Border on the Content Containers
- Set Margin on the Content Containers

Test using your fix, with the `display: none;` that might solve the porblem.

Reply:
Also to think about:
If you set padding to 0, and margin to 0, and there is a transition set on the Content Container, then when the `.e-active` is removed, it will animate from 0 to 20px for the set transition time. So we could even take of the `padding: 0` if the display none works.

Reply:
Also,
Seeing as this is not an issue on Desktop, maybe we should nest it in the `.e-collapse`, so this fix only effect responsive.

Reply:
> Without the display none, see what I'm getting ![image](https://user-images.githubusercontent.com/82999080/223703566-5cde53dc-92d8-4a55-8d27-7330ef26dcbf.png)
> 
> * Set Border on the Content Containers
> * Set Margin on the Content Containers
> 
> Test using your fix, with the `display: none;` that might solve the porblem.

@StuartElementor 

`display: none` is causing problems. 

I can fix the border and margin issues.

Reply:
> 

Will do.

Reply:
@StuartElementor 

I have assigned my styling to the accordion version only. 
And I have removed all styling from not active containers.